lib/rialto/etl/logging.rb

```diff
@@ -12,9 +12,11 @@ module Logging
       def logger
         @logger ||= Yell::Logger.new(:null).tap do |logger|
           # Everything to stderr
-          logger.adapter :stderr
+          # logger.adapter :stderr
```

Comment:
Cruft to be cleaned up later, but I suspect this was already on your to-do list!